drivers/bluetooth/hci/hci_ifx_cyw208xx.c

```diff
@@ -165,11 +165,16 @@ static int cyw208xx_bt_firmware_download(const uint8_t *firmware_image, uint32_t
 	return 0;
 }
 
+#define HCI_BRCM_SET_LOCAL_DEV_ADDR     (0xFC01)
+#define BTM_SET_LOCAL_DEV_ADDR_LENGTH   6
+
+
```

Comment:
minor: remove one of these empty lines (there should never be a need to have two consecutive ones)